app/Controllers/configureController.php

```diff
@@ -241,13 +241,16 @@ public function archivingAction() {
 	 * checking if categories and feeds are still in use.
 	 */
 	public function queriesAction() {
```

Comment:
Don't you think query-related action could be moved into a dedicated controller? It's just an idea but it can help to reduce `configureController` file size.